src/views/ActorProjects/ActorProjectsView.tsx

```diff
@@ -226,3 +228,10 @@ const TextNotFound = styled('p')(({ theme }) => ({
     lineHeight: '38.4px',
   },
 }));
+
+const TeamHeaderContainer = styled('div')(({ theme }) => ({
+  marginTop: 104,
+  [theme.breakpoints.up('tablet_768')]: {
+    marginTop: 0,
+  },
+}));
```

Comment:
Please, note that this is handled in the `PageWrapper` component, maybe just using `PageContainer` rather than `PageWrapper` solves the issue, but if not, move the css to that component instead of wrapping the team header

Reply:
You are right, done